src/test/java/org/mastodon/mamut/clustering/ui/DendrogramPanelTest.java

```diff
@@ -146,6 +163,26 @@ public void testGetVerticalLine()
 		assertEquals( 312d, line.getX1(), 0.01 );
 	}
 
+	@Test
+	public void testExport() throws IOException
+	{
+		DendrogramPanel< String > dendrogramPanel = new DendrogramPanel<>( classification );
+		int width = 600;
+		int height = 600;
+		Image image = new BufferedImage( width, height, BufferedImage.TYPE_INT_ARGB );
```

Comment:
This `image` is not used in the test. Please remove it.

Reply:
The image itself is not used, but the graphics context derived from it. Thus I need to keep it.